client/src/main/resources/swaggercodegen/libraries/jersey2/JSON.mustache

```diff
@@ -1,62 +0,0 @@
-package {{invokerPackage}};
```

Comment:
Why is this deleted? Don't we need it to configure the object mapper behavior?

Reply:
This was added erroneously when we thought Jersey 3 would work with all the other Terra services importing the TPS client. We were wrong, so this is no longer needed.